script/interfaces/ICreate3Factory.sol

```diff
@@ -0,0 +1,22 @@
+// SPDX-License-Identifier: AGPL-3.0
```

Comment:
i removed the submodule for create3-factory its not really used beyond interface so i just added it here to keep the build simple